product-matrix.json

```diff
@@ -37,10 +36,11 @@
       "comments": "IntelliJ 2019.2 EAP",
       "name": "IntelliJ 2019.2 EAP",
       "version": "2019.2",
+      "isTestTarget": "true",
       "ideaProduct": "ideaIU",
       "ideaVersion": "2019.1",
       "dartPluginVersion": "191.4899",
-      "sinceBuild": "192.0",
+      "sinceBuild": "191.0",
```

Comment:
I think we'll over overlap issue wrt the version ranges that match each plugin (2019.1 vs 2019.2). Is this something we'd want to land? Something to enable testing?